ionoscloud/resource_nic.go

```diff
@@ -334,7 +334,7 @@ func NicSetData(d *schema.ResourceData, nic *ionoscloud.Nic) error {
 				return fmt.Errorf("error setting ips %w", err)
 			}
 		}
-		if nic.Properties.Ipv6Ips != nil && len(*nic.Properties.Ipv6Ips) > 0 {
+		if nic.Properties.Ipv6Ips != nil {
```

Comment:
@cristiGuranIonos above we have `nic.Properties.Ips != nil && len(*nic.Properties.Ips) > 0`, why is it different here?

Reply:
@cristiGuranIonos I discussed with @cbeti-ionos and now it is clear, but please add a comment here to make things clearer for the others.

Reply:
added